frontend/src/components/Navbar.js

```diff
@@ -0,0 +1,27 @@
+import React, { useState } from 'react';
```

Comment:
The state hook is no longer being used either do you can delete the useState part and just keep the React part like this import React from 'react';